packages/backend/server/src/core/workspaces/resolvers/workspace.ts

```diff
@@ -383,19 +382,20 @@ export class WorkspaceResolver {
     @CurrentUser() user: CurrentUser,
     @Args('workspaceId') workspaceId: string,
     @Args('email') email: string,
-    @Args('permission', { type: () => Permission }) permission: Permission,
-    @Args('sendInviteMail', { nullable: true }) sendInviteMail: boolean
+    @Args('sendInviteMail', { nullable: true }) sendInviteMail: boolean,
+    @Args('permission', {
+      type: () => Permission,
+      nullable: true,
+      deprecationReason: 'never used',
+    })
+    _permission?: Permission
   ) {
     await this.permissions.checkWorkspace(
       workspaceId,
       user.id,
       Permission.Admin
     );
 
-    if (permission === Permission.Owner) {
-      throw new CantChangeSpaceOwner();
```

Comment:
why remove the restriction for inviting?

Reply:
<img width="485" alt="image" src="https://github.com/user-attachments/assets/a15ea72e-227a-48e1-b9e2-a2e70ba4eb2e" />

normal workspace also support transfer ownership

Reply:
https://github.com/toeverything/AFFiNE/blob/db61123b4fedbc9cde7ce94d07bcf11635306bb3/packages/backend/server/src/core/workspaces/resolvers/workspace.ts#L428-L432

invite no longer relies on permissions provided by the client, so there is no need to check